### ISSUE TITLE
Json path of the copied value

### ISSUE THREAD

**Issue by davidsu06** [mac-s-g/react-json-view#306]
Hi, how could I get the json path of the copied value? I see it can be done with the namespace that returns 'onSelect', but this obligates to click the value. Is it possible to do it with the 'enableClipboard' callback?

I need to copy the path when the user clicks the icon that just 'enableClipboard' shows.

Thanks for your help.

**Comment by michelnev** [comment]
The `enableClipboard` callback does return the namespace.  However, the namespace is incorrect in v1.19.1, I'm about to try out the latest release to see if it has been fixed.  It returns null in the first element and if the top element is clicked the `name` and `namespace` are null.  For example, when copying a root level element, the following is returned:
```
{
  "src": "the value is correct",
  "namespace": [
    null
  ],
  "name": null
}
```

**Comment by mac-s-g** [comment]
Just tested this out and the bug still exists.  This was reported first by #305 so I'm gonna close this as a dupe issue.  Let's use 306 to track progress on the resolution.

**Comment by mac-s-g** [comment]
#321 resolves the issue. once tests pass it will be published to 1.20.4
